docs/conf.py

```diff
@@ -398,8 +399,10 @@ def setup(app: Sphinx) -> Dict[str, Any]:
     r"https://gitlab.com": "https://about.gitlab.com/",
     r"http://www.yahoo.com": "https://www.yahoo.com/",
     r"https://feature-engine.readthedocs.io/": "https://feature-engine.trainindata.com/en/latest/",
+    r"https://picsum.photos/": "https://fastly.picsum.photos/",
 }
 
 # we have had issues with linkcheck timing and retries on www.gnu.org
 linkcheck_retries = 3
-linkcheck_timeout = 50
+linkcheck_timeout = 30
+linkcheck_anchors = False
```

Comment:
Would you like to still check anchors when a PR is merged, but not on creation or update? Then it wouldn't slow down reviews, but it would ultimately catch problems.

Reply:
I am on the fence here. It would be nice to catch anchor issues, but before removing them, we had some CI jobs running for 6 hours(!), which is a lot of CI hours for a link-checking job (and a lot of compute resources 🌲🔥).

Ref: https://github.com/pydata/pydata-sphinx-theme/actions/workflows/docs.yml

Reply:
In Plone docs, we used to have high limits. Now we use these.

```python
linkcheck_report_timeouts_as_broken = True
linkcheck_retries = 1
linkcheck_timeout = 5
```

This dramatically reduced compute time, and has not resulted in false positives. This configuration avoids the noise of extremely brief outages of 5 seconds or fewer, while allowing one retry. The reported failures almost always are extended outages or redirects. I joke with my colleagues that linkcheck is the best server monitoring service ever!

Reply:
@trallard, do we want to adopt these rules?

Reply:
Yep, I like them. Will make changes later.